### PR TITLE
Add new updates to the ALCF workflow

### PR DESCRIPTION
**Pull request by aisclark91**
I updated the needed workflow to work on Polaris, without issues.

### CODE REVIEW

File: sphinx_docs/source/alcf.rst [hunk 1]
```diff
@@ -108,3 +111,42 @@ A script that can be used to chain jobs with PBS is:
...
+   conda activate
+   VENV_DIR="venvs/polaris"
+   source "${VENV_DIR}/bin/activate"
+
+Once the virtual environment is active, any extra package can be installed with
+the use of ``pip``:
+
+.. prompt:: bash
+
+   python pip -m install <module>
```
Comment (yut23) [inline sphinx_docs/source/alcf.rst:152]:
```suggestion
   python -m pip install <package>
```

File: sphinx_docs/source/alcf.rst [hunk 2]
```diff
@@ -108,3 +111,42 @@ A script that can be used to chain jobs with PBS is:
...
+
+.. prompt:: bash
+
+   module use /soft/modulefiles
+   module load conda 
+   conda activate
+   VENV_DIR="venvs/polaris"
+   mkdir -p "${VENV_DIR}"
+   python -m venv "${VENV_DIR}" --system-site-packages
+   source "${VENV_DIR}/bin/activate"
```
Comment (yut23) [inline sphinx_docs/source/alcf.rst:135]: We should mention these commands come from https://docs.alcf.anl.gov/polaris/data-science-workflows/python/, so we know where to look if this stops working

File: sphinx_docs/source/alcf.rst [hunk 3]
```diff
@@ -52,6 +51,10 @@ https://www.alcf.anl.gov/support/user-guides/polaris/queueing-and-running-jobs/j
 
 For production jobs, you submit to the ``prod`` queue.
 
+For debugging jobs, there are two options: the ``debug`` queue for only one node use jobs, and
```
Comment (yut23) [inline sphinx_docs/source/alcf.rst:54]: `debug` allows 1 to 2 nodes per job, with a max of 24 nodes in use across the entire system; `debug-scaling` allows 1 to 10 nodes per job, with a max of 1 job in any state per user (see https://docs.alcf.anl.gov/polaris/running-jobs/#queues)

Comment (aisclark91) [reply sphinx_docs/source/alcf.rst:54]: That seems to be recent, I will correct this but I will not mention the max of 24 nodes; I will refer to the webpage instead.

File: sphinx_docs/source/alcf.rst (hunk 2 above)
Comment (aisclark91) [reply sphinx_docs/source/alcf.rst:133]: I mentioned it!

**Review by yut23 — APPROVED** [review]

**Pull request merged**